packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenuInput.tsx

```diff
@@ -22,13 +22,13 @@ const StyledInput = styled.input<{
   position: relative;
   width: 100%;
 
-  &:focus {
+  /* &:focus {
```

Comment:
Don't be afraid to delete code, that's what git is for - we can always find it if we need it! So don't leave commented code in the codebase :)